pkg/project/project.go

```diff
@@ -263,7 +262,6 @@ func FindFileOrDirectory(directory, filename string) (string, bool) {
 func isRootPath(directory string) bool {
 	return (directory == "." ||
 		directory == string(filepath.Separator) ||
-		directory == "" ||
```

Comment:
This won't be the fix, since it can only make it stop iterating less often not more often.